### ISSUE TITLE
The CinderVolume @JsonProperty error

### ISSUE THREAD

**Issue by tangxiaodon89** [gondor/openstack4j#402]
In the Class org.openstack4j.openstack.storage.block.domain.CinderVolume the property 'name','description' now is.

![qq 20150603101149](https://cloud.githubusercontent.com/assets/11000963/7951354/fd582f48-09d8-11e5-8aad-6ab8cd53d386.jpg)

The openstack API has changed it.
![image](https://cloud.githubusercontent.com/assets/11000963/7951368/26205180-09d9-11e5-8439-10fd9a6a4058.png)

So i change the class to this.
![image](https://cloud.githubusercontent.com/assets/11000963/7951377/3a362870-09d9-11e5-9484-4b33501fb9f9.png)

**Comment by gondor** [comment]
Yes you are correct for the V2 based endpoint, however V1 is display_name, display_description.  I'm working on a solution to provide support with the same return type regardless of V1 or V2.  This issue is the one we're tracking against: https://github.com/gondor/openstack4j/issues/395

**Comment by tangxiaodon89** [comment]
thanks
